db/model/room.js

```diff
@@ -176,6 +176,13 @@ module.exports = function room(seq, dataTypes) {
       through: 'RoomWriters',
       foreignKey: 'roomId',
     });
+    Room.addScope('namespace', {
+      attributes: ['id', 'name'],
+      where: {
+        active: true,
+        isDeleted: 0,
```

Comment:
Oops--none of us caught that the Room model does not have an "isDeleted" attribute!!! Fixing in https://github.com/salesforce/refocus/pull/960